lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -46,7 +47,7 @@ internal sealed class TaskRuntime : IObserver<ICloseEvent>, IObserver<ISuspendEv
         [Inject]
         private TaskRuntime(
```

Comment:
Agreed, I'll fix it. It seems like we have a lot of trailing spaces for some reason.